### ISSUE TITLE
Specification only has IO intensive Actions

### ISSUE THREAD

**Issue by tectrixer** [umgefahren/server-language-benchmark#8]
# Current Problem
Right now all actions required in the specification are IO bound tasks. The most compute intensive task is the parsing of the incoming command.

# Solution proposal
Add some of the following actions, they do not require much developing time but also benchmark computation capability:
- `GETKN key-pattern` get the number of keys containing the given key pattern at least once
- `GETVN value-pattern` get the number of values containing the given value pattern at least once

**Comment by tectrixer** [comment]
YES

**Comment by umgefahren** [comment]
We've discussed introducing a task fit for your requirement of a computing intensive task. We suggest that prime factorization would be a good task, since it's NP-complete and easily verifiable by the client.

We'll specify a specific algorithm that has to be implemented in order to achieve fairness (this problem can be heavily optimized).

However there are still some issues to resolve before we start working on this, so it has a lower priority, but will be added to the spec soon.

Sadly CPU intensive tasks like this clearly favor compiled, procedural languages over interpreted, functional languages thus, it's not that fit for a compelling server benchmark.
